Source/Scene/Cesium3DTileset.js

```diff
@@ -786,6 +802,68 @@ define([
 
     ///////////////////////////////////////////////////////////////////////////
 
+    var scratchJulianDate = new JulianDate();
+
+    function updateExpireDate(tile) {
+        if (defined(tile.expireDuration)) {
```

Comment:
If `expire.date` is defined, it is set in the `Cesium3DTile` constructor. Here it sees if the duration would be less than the stored date.